src/validation.cpp

```diff
@@ -1205,20 +1215,35 @@ PackageMempoolAcceptResult MemPoolAccept::AcceptMultipleTransactions(const std::
         m_viewmempool.PackageAddTransaction(ws.m_ptx);
     }
 
+    // Transactions must meet two minimum feerates: the mempool minimum fee and min relay fee.
+    // For transactions consisting of exactly one child and its parents, it suffices to use the
+    // package feerate (total modified fees / total virtual size) to check this requirement.
+    const auto m_total_vsize = std::accumulate(workspaces.cbegin(), workspaces.cend(), int64_t{0},
```

Comment:
This assumes that `AcceptMultipleTransactions` always takes proper packages and that we can safely consider the aggregated feerate of the bunch of transactions. It's not the case.
https://github.com/bitcoin/bitcoin/blob/e3c08eb620a2f317fc09fdf20969fa26f02afb91/src/validation.cpp#L555-L560

This patch demonstrates how this makes `testmempoolaccept` accept invalid transactions (by considering the package feerate for unrelated transactions):
```patch
diff --git a/test/functional/mempool_accept.py b/test/functional/mempool_accept.py
index d3961e753d..f8dd380ee5 100755
--- a/test/functional/mempool_accept.py
+++ b/test/functional/mempool_accept.py
@@ -333,6 +333,15 @@ class MempoolAcceptanceTest(BitcoinTestFramework):
             maxfeerate=0,
         )
 
+        self.log.info("Don't consider package fees between unrelated transactions")
+        txs = [
+            self.wallet.create_self_transfer(fee_rate=0, mempool_valid=False)['tx'],
+            self.wallet.create_self_transfer()['tx'],
+        ]
+        res = self.nodes[0].testmempoolaccept([tx.serialize().hex() for tx in txs])
+        # The first tx has a feerate of 0. No child pay for it. It should be rejected.
+        assert not res[0]["allowed"]
+
 
 if __name__ == '__main__':
     MempoolAcceptanceTest().main()
```